web-registry/src/components/organisms/CreditBatches.tsx

```diff
@@ -25,19 +25,20 @@ import { truncateWalletAddress } from '../../lib/wallet';
 import { ledgerRestUri } from '../../ledger';
 import { getBatchSupply, getBatches } from '../../lib/ledger-rest';
 import { getAccountUrl } from '../../lib/block-explorer';
+import { BatchRowData } from '../../types/ledger';
 
-interface BatchRowData {
-  start_date: string | Date;
-  end_date: string | Date;
-  issuer: string;
-  batch_denom: string;
-  class_id: string;
-  total_amount: number;
-  tradable_supply?: number;
-  retired_supply?: number;
-  amount_cancelled: number;
-  project_location: string;
-}
```

Comment:
Moved to shared `/ledger` types

Reply:
I did something similar. I actually changed the `ledger-rest` file to `ecocredit`.

Reply:
I like how you made a types folder, but maybe it should be called `ecocredit`?

Reply:
I'm fine with that! Will update 👍